d3rlpy/algos/qlearning/base.py

```diff
@@ -360,6 +360,8 @@ def fit(
         n_steps_per_epoch: int = 10000,
         experiment_name: Optional[str] = None,
         with_timestamp: bool = True,
+        logging_steps: int = 500,
+        logging_strategy: str = "epoch",
```

Comment:
This is a good idea! I have one request to change how we switch `epoch` and `steps`. I'd like to use Enum to specify this option. In this way, could you add `LoggingStrategy` enum class to `constants.py` as follows?
https://github.com/takuseno/d3rlpy/blob/06dd7a7698bf68af0645afeb62f148aad921a9bd/d3rlpy/constants.py#L47

Then, it's easier for users to see what kind of options are available.